documentation/src/main/asciidoc/topics/rest_api_v2.adoc

```diff
@@ -406,6 +406,201 @@ To obtain a list of caches available in a {brandname} cluster, invoke a `GET` re
 include::rest_examples/get_v2_list_caches.adoc[]
 ----
 
+[[rest_v2_cache_xsite]]
+=== Cross site replication
+
+Use the REST API to monitor and control Cross Site (x-site) replication on your {brandname} cluster.
+
```

Comment:
just wondering if it is useful a link to the xsite docs.

Reply:
+1

Reply:
I am trying to insert a simple link to other page for 10min without success, what is the convoluted way of doing it?  :)

Reply:
I think it worked, after several trial and errors :+1: